packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js

```diff
@@ -49,7 +47,7 @@ export function NativeValue({ data, edit, onSelect, onChange, jsonpath, selected
 			type="button"
 			onClick={e => stopAndSelect(e, { onSelect, jsonpath })}
 		>
-			{display}
+			<span className={theme['inner-button-text']}>{display}</span>
```

Comment:
It seems that this span is not needed.
You added the css to make the ellipsis on the button and this span.
--> remove this extra span please